src/HttpCache/EventListener/AddTagsListener.php

```diff
@@ -66,6 +66,13 @@ public function onKernelResponse(FilterResponseEvent $event)
             return;
         }
 
+        // Encode tags for greater compatiblity with different proxies
+        // Some do not allow special characters like / or @ in cache tags and
+        // also it allows to use a , in a tag, if you wish to do so.
+        $resources = array_map(function($resource) {
```

Comment:
We were using md5 initially, we switched back to plain text because it is very hard to debug when reading/grepping logs. I would prefer to encode special char at the cache adapter layer.

Reply:
I can do that but to remain BC I need to introduce another interface for the Purgers that implement both PurgerInterface and...hmmm... CacheTagsFormatterInterface?